utils/util.js

```diff
@@ -0,0 +1,24 @@
+export const formatPhone = phone => {
+  if (!phone) return;
+  if (phone[0] === '(') return phone;
+
+  let newNumbers = '';
+  const format = phone.split('-');
+  for (let i = 0; i < format.length; i++) {
+    const currentDigits = format[i];
+
+    if (i === 0) {
+      newNumbers = `(${currentDigits}) `;
+    } else if (i === 1) {
+      newNumbers = newNumbers + `${currentDigits}-`;
+    } else {
+      newNumbers = newNumbers + `${currentDigits}`;
+    }
+  }
+
+  return newNumbers;
```

Comment:
I'd probably take an approach of stripping the string of all non-digit characters first (using regex) and then formatting in the desired format

Reply:
Makes sense. Once again, if I thought to use regex it would have been a lot neater and will cover a lot more cases.